src/routing/RouterContext.tsx

```diff
@@ -26,6 +47,8 @@ export const routeObjectsReducer = (routeObjectsArray: RouteObject[]) => {
           title: (
             (current?.element as ReactElement)?.props as { title: string }
           )?.title,
+          conditional: conditionalPath,
+          isShown: conditionalPath && field && field[0].value === true,
```

Comment:
Just making sure I understand. Is it true that the `isShown` property here will only be used when looking at a conditional route, as it looks like it'll default to `false` on a non-conditional route? Like the example in `RouterProgress`

Reply:
yes, it should be false or null on a non-conditional route. I think there could be some clean up on this so that only one property is used for this condition (conditional routes). The reason why I have to check for true is because the field sometimes defaults to all of the values in formik, and I can't just null out `isShown` like typical variables because its a react hook.